src/components/SimpleInput.js

```diff
@@ -0,0 +1,55 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
```

Comment:
My first thought was if we wanted to restrict the type prop to only valid input types, but then I saw the list.

It makes me curious though about how this would handle a type of `checkbox` or even `file`